### DISCUSSION TITLE
Cant login to my code-server

### DISCUSSION THREAD

**Discussion by Kaushal-Kishore-Gupta** [coder/code-server#5066] [Q&A]
I installed my cod-server in digital ocean and set my password here
sudo nano /lib/systemd/system/code-server.service

and here .config/code-server/config.yaml

in both the files, I set my password the same but every time I try to log in it says incorrect password please help me

**Comment by lg-nineoneninethreenine** [comment]
Did you rebuild the container? I think @jsjoeio told about rebuilding containers when you update crucial files. Try that and report back :)

**Reply by jsjoeio** [reply]
Yeah, I think this should do it. Please rebuild your environment!

**Comment by Kaushal-Kishore-Gupta** [comment]
No i set it up from scratch following https://www.digitalocean.com/community/tutorials/how-to-set-up-the-code-server-cloud-ide-platform-on-ubuntu-18-04

**Comment by lg-nineoneninethreenine** [comment]
> No i set it up from scratch following https://www.digitalocean.com/community/tutorials/how-to-set-up-the-code-server-cloud-ide-platform-on-ubuntu-18-04

@jsjoeio can you try helping him in an easy way? I don't understand, seeing one of your previous replies, I logged that you have to rebuild containers to switch passcodes. Could be wrong, but if anyone is a code-server pro with digital ocean, they could chime in.

**Reply by jsjoeio** [reply]
Let's wait for them to reply and see if they need more help
